generator/src/Pos/Generator/Block/Mode.hs

```diff
@@ -21,7 +21,7 @@ module Pos.Generator.Block.Mode
 import           Universum
 
 import           Control.Lens.TH (makeLensesWith)
-import qualified Control.Monad.Catch as UnsafeExc
+import qualified Control.Monad.Catch as UncheckedExc
```

Comment:
Seriously?

Reply:
Whoops, I checked the `grep` output, but missed those. Sorry!